docs/Relays.md

```diff
@@ -0,0 +1,27 @@
+AutoFixture is focused on creating instances of concrete types. For this reason it doesn't have any support for non-concrete types like interfaces and abstract classes.
+
+Most of the time, AutoFixture needs to resort to other libraries, like Moq, to create objects implementing an interface or representing a dummy specialization of an abstract class.
+
+Sometimes, it's sufficient to return a concrete type whenever a non-concrete type is requested. To handle cases like this, AutoFixture offers the class `TypeRelay`.
+
+A `TypeRelay` is a customization that can be fed to the Fixture that simply maps relays requests of a certain type to another one.
```

Comment:
I guess either the word `maps` or `relays` should be removed from this sentence?

Reply:
Do you have a suggestion? I couldn't come to anything better

Reply:
oh... I see what you meant